### DISCUSSION TITLE
Magma Sentry - Invalid -I Flag

### DISCUSSION THREAD

**Discussion by FrancescoVall** [magma/magma#15339] [Q&A] [answered]
Hello all.
I'm really new to this so i hope use the correct words.
I'm installing, following Quick Start Guide the Magma Core. But when i  ssh into magma and the launch with make run i encontered
the following:
[10/30] Performing build step for 'MagmaSentry'
FAILED: MagmaSentry-prefix/src/MagmaSentry-stamp/MagmaSentry-build 
cd /magma_common/sentry && /usr/bin/cmake --build .
[1/6] Running C++ protobuf compiler on /orc8r/protos/common.proto
FAILED: orc8r/protos/common.pb.h orc8r/protos/common.pb.cc 
cd /magma_common/sentry && /usr/bin/protoc -I /orc8r/protos/prometheus -I -I --cpp_out=/magma_common/sentry /orc8r/protos/common.proto
Missing value for flag: -I
[2/6] Running C++ protobuf compiler on /orc8r/protos/mconfig/mconfigs.proto
FAILED: orc8r/protos/mconfig/mconfigs.pb.h orc8r/protos/mconfig/mconfigs.pb.cc 
cd /magma_common/sentry && /usr/bin/protoc -I /orc8r/protos/prometheus -I -I --cpp_out=/magma_common/sentry /orc8r/protos/mconfig/mconfigs.proto
Missing value for flag: -I
ninja: build stopped: subcommand failed.
[11/30] No install step for 'ServiceRegistry'
ninja: build stopped: subcommand failed.
make: *** [Makefile:109: build_common] Error 1

I really try to debug this, and seems the multiple -I -I in the command cause the issue.
But i 'm not able to solve.
Anyone can help me?

Thanks in Advance.
Francesco.

**Comment by jordanvrtanoski** [comment]
Can you share the steps that you are using?

**Comment by panyogesh** [comment, marked as answer]
Marking at as closed as its inactive from long time.
